_includes/right_sidebar.html

```diff
@@ -1,11 +1,8 @@
 <div id="right-sidebar" class="sidebar">
-  <a href="#" class="right-sidebar-h2">True Privacy with Circuits</a>
-  <a href="#" class="right-sidebar-h3">Distributed Ledger Platform as a Service</a>
-  <a href="#" class="right-sidebar-h2">Flexibility with Splinter’s Modular Architecture</a>
-  <a href="#" class="right-sidebar-h2">Smart Contracts That Can Share State</a>
-  <a href="#" class="right-sidebar-h2">Quick Startup for Application Development</a>
-  <a href="#" class="right-sidebar-h3">Event Subscription for State Changes</a>
-  <a href="#" class="right-sidebar-h3">Complex Patterns for Data Flow</a>
-  <a href="#" class="right-sidebar-h2">Auditable Open Source</a>
-  <a href="#" class="right-sidebar-h3">Why NOT Splinter?</a>
+  <p id="right-sidebar-list"></p>
+
+  <script src="/scripts/generateRightSidebar.js"></script>
```

Comment:
IMO we should avoid any inline scripts and keep all JS usage in one place; currently that's `_includes/scripts.html`.

Reply:
moved